example/basic/main.go

```diff
@@ -55,11 +57,9 @@ func main() {
 		tag.Insert(barKey.String("bar1")),
 	)
 
-	gauge := meter.GetFloat64Gauge(
-		ctx,
-		oneMetric,
-		lemonsKey.Int(10),
-	)
+	gauge := oneMetric.Handle(ctx, meter, lemonsKey.Int(10))
```

Comment:
Shouldn't this also use `fooKey` and `barKey`? Or maybe it would be good to describe a comment what happens here if those keys are missing.

Reply:
Good catch. I need to update the RFC about this topic, it was discussed heavily in the working session.